src/back/bootstrap/index.js

```diff
@@ -9,10 +9,18 @@ const
 
 module.exports = (app) => {
     db(formatMongooseDbUriFromEnv(env));
+    extendRequest();
     preMiddleware(app);
     return app.use(controllers);
 };
 
+function extendRequest() {
+    const authService = require('../controllers/auth/auth.service'),
+        http = require('http');
+    http.IncomingMessage.prototype.getCurrentUser = authService.getCurrentUser;
```

Comment:
On the other hard [Express doesn't](https://github.com/expressjs/express/issues/1205#issuecomment-6748051). Let's see what other members say. If everybody is ok with this solution I don't mind it is merged.